docs/lib/analytics/fragments/ios/record.md

```diff
@@ -41,35 +40,36 @@ Events have default configuration to flush out to the network every 60 seconds.
 }
 ```
 
-> **Note**: If you set `autoFlushEventsInterval` to 0, you are responsible for calling `Amplify..flushEvents()` to flush events.
+> **Note**: If you set `autoFlushEventsInterval` to 0, you are responsible for calling `Amplify.Analytics.flushEvents()` to submit the recorded events to the backend.
 
 
 ## Global Properties
 
-You can register properties which will be used across all `Amplify.Analytics.record`.
+You can register properties which will be used across all `Amplify.Analytics.record(event:)` calls.
 
 ```swift
-let globalProperties = ["globalPropertyKey": "value"] as [String: AnalyticsPropertyValue]
+let globalProperties: [String: AnalyticsPropertyValue] = ["globalPropertyKey": "value"]
 Amplify.Analytics.registerGlobalProperties(globalProperties)
 ```
 
-To unregister all global properties call `Amplify.Analytics.unregisterGlobalProperties()`. 
+To unregister all global properties call `Amplify.Analytics.unregisterGlobalProperties()`:
```

Comment:
What you are saying and imply here are different things:

- Calling `Amplify.Analytics.unregisterGlobalProperties()` will remove all global properties
- *Additionally*, you can remove one or more properties by name by passing a list

This could use further clarification as the colon implies one follows the other.